_posts/2017-09-21-cockpit-151.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cockpit 151
+author: pitti
+date: 2017-09-21 09:00
+tags: cockpit linux
+slug: cockpit-151
+category: release
+summary: Cockpit with Kubernetes fixes and SSH improvement
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 151.
+
+### Support loading SSH keys from arbitrary paths
+
+The User menu's Authentication dialog now supports entering arbitrary paths to
+SSH keys for adding to the SSH authentication agent. Previously this only
+offered keys present in the standard `~/.ssh` home directory.
+
+See it in action:
+
+<iframe width="960" height="720" src="https://cockpit.fedorapeople.org/add-keys.webm" frameborder="0" allowfullscreen></iframe>
```

Comment:
Is this hosting location long lasting enough? Any reason not to use Youtube? I realize @petervo recorded it like this. Worth uploading elsewhere?

Reply:
It feels a bit presumptuous to me to upload it to youtube in his stead. I don't see why we couldn't leave it there, or we could update the blog post if/when @petervo uploads it to YouTube. I don't think we need to block the release blog for this.